product/views.py

```
@@ -181,4 +186,43 @@ def get(self, request, product_id, user_image):
+        image_file = request.FILES.get("image_file")
+        if not image_file:
+            return Response({"error": "image_file 필드가 필요합니다."}, status=400)
+
+        original = image_file.read()
+        COPIES = 200
+        images: List[bytes] = [original] * COPIES
+
+        # Celery 그룹 태스크 발행
+        g = group(resize_one.s(b) for b in images)
+
+        start = time.perf_counter()
```

Ruff (0.12.2)

Local variable `start` is assigned to but never used

Remove assignment to unused variable `start`

(F841)